src/main/java/com/contentful/java/cda/CDALocale.java

```diff
@@ -10,6 +10,9 @@
 
   String name;
 
+  @SerializedName("fallbackCode")
+  String fallbackLocaleCode;
```

Comment:
Yes.

I wanted to keep it aligned with the naming of the next field: `default` from Contentful gets renamed to `defaultLocale` to identify what it is the default of, so I thought it makes also sense for identifying what it is the `fallbackCode` for ....